src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -19,6 +19,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Inherits AbstractSyntaxFactsService
         Implements ISyntaxFactsService
 
+        Public Shared ReadOnly Property [Default] As New VisualBasicSyntaxFactsService
```

Comment:
You need to remove the `<ExportLanguageService>` once you have an `<ExportLanguageServiceFactory>`.